src/infi/clickhouse_orm/database.py

```diff
@@ -79,6 +82,56 @@ def gen():
                 yield ('\n'.join(batch) + '\n').encode('utf-8')
         self._send(gen())
 
+    def insert_try_best(self, model_instances, batch_size = 1000, is_create = False, insert_count = 0, first_recu = True):
+        cache = []
+        table_name = None
+        for elem in model_instances:
+            if is_create == False:
```

Comment:
here you're using variable is_create and then make `create_table` if `is_create == False`. This seems to be non-obvious behaviour. `is_create` sounds more like something should be created, but you're doing it vise-versa. Please choose another name for variable or change logic. In general, purpose of this recursive method is not clear for me now. Please describe this code.